Library/Formula/go.rb

```diff
@@ -76,9 +93,10 @@ def caveats; <<-EOS.undent
     `go vet` and `go doc` are now part of the go.tools sub repo:
       http://golang.org/doc/go1.2#go_tools_godoc
 
-    To get `go vet` and `go doc` run:
+    `godoc` has been installed for you unless you requested otherwise.
+
+    To get `go vet` run:
       go get golang.org/x/tools/cmd/vet
-      go get golang.org/x/tools/cmd/godoc
```

Comment:
Had some fun with ` vet `. Turns out Go doesn't install that into the bin, ever. It's placed 4 directories deep inside ` libexec `, inside a special ` tools ` directory. Placing it anywhere else, or even symlinking it into the bin and then calling it directly = so much breakage :smile_cat:. Working now though, added a note that it shouldn't be symlinked to the bin despite the weird location. Pushing now.